include/depthimage_to_laserscan/DepthImageToLaserScan.hpp

```diff
@@ -69,13 +71,19 @@ class DEPTHIMAGETOLASERSCAN_EXPORT DepthImageToLaserScan final
    *                    radii for each angular increment.  The output scan will output the closest radius that is
    *                    still not smaller than range_min.  This can be used to vertically compress obstacles into
    *                    a single LaserScan.
+   * @param quantile_value The quantile value to use for calculating the distance for each column.
+   *                       This value determines which distance measurement to use from the multiple
+   *                       rows of depth data. For example, a quantile value of 0.1 will use the 10th
```

Comment:
I feel the second sentence says the same information as the introduction but in a less direct way.  A column is a more conceptually clear concept and is the standard language.